.npmignore

```diff
@@ -1,7 +1,9 @@
 node_modules/
-.idea/
 npm-debug.log
 coverage
 docs
+spec
 local_testing
-*.md
\ No newline at end of file
+LICENSE.md
+CONTRIBUTING.md
```

Comment:
I think you actually do want to publish the license and contrib info along with your npm package